mart/views.py

```diff
@@ -22,4 +22,22 @@ def welcome(request):
 
 def display_products(request):
     active_products = Product.objects.filter(isActive=True)
-    return render(request, 'mart/products.html', {'products': active_products})
\ No newline at end of file
+    return render(request, 'mart/products.html', {'products': active_products})
+
+def cart(request):
+    selected_products = request.GET.getlist('products')
+    products = {
+        "sunglasses": 500,
+        "microwave": 700,
+        "tshirt": 230,
+    }
+    cart_products = []
+    total = 0
+    for product in selected_products:
+        if product in products:
```

Comment:
Where is the serializer part?

Reply:
It has been added, you may review now :)